src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/Product/Combination/bulk.html.twig

```diff
@@ -25,14 +25,14 @@
 {% set legacyBaseLayout = 'light_display_layout.tpl' %}
 {% extends '@PrestaShop/Admin/layout.html.twig' %}
 
-{% form_theme bulkCombinationForm '@PrestaShop/Admin/Sell/Catalog/Product/FormTheme/bulk_combination.html.twig' %}
+{% form_theme bulkCombinationForm '@PrestaShop/Admin/Sell/Catalog/Product/FormTheme/combination.html.twig' %}
 
 {% block stylesheets %}
   <link rel="stylesheet" href="{{ asset('themes/new-theme/public/product' ~ rtl_suffix ~ '.css') }}" type="text/css" media="all">
 {% endblock %}
 
 {% block content %}
-  {{ form_start(bulkCombinationForm) }}
+  {{ form_start(bulkCombinationForm, {'attr': {'class': 'combination-page' }}) }}
```

Comment:
Careful I tested the PR and this removes the initial `bulk-combination-form` that was set on the form thus removing a CSS rule on it and breaking the design

Reply:
```suggestion
  {%- set attr = attr|merge({'class': (attr.class|default('') ~ ' combination-page')|trim }) -%}
  {{ form_start(bulkCombinationForm) }}
```

Reply:
This should fix it

Reply:
Hmm, then maybe it's easier and cleaner to set `combination-page` on the form type